content/dojos/69-online.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "#69 Online"
+date: 2020-10-02  T0:0:0+01:00
```

Comment:
Onverwachtte spaties voor de T, maar het lijkt er op dat Hugo er geen last van heeft en de pagina in de lijst van Dojo's toch goed gesorteerd is...